q2_quality_filter/tests/test_filter.py

```diff
@@ -255,10 +255,10 @@ def test_process_record(self):
         self.assertEqual(status, exp_status)
 
     def test_is_retained(self):
-        filtering_stats_df = pd.DataFrame(
+        filtering_stats = pd.Series(
```

Comment:
Could you add a test that calls `q_filter` with `num_processes=2`, and confirm that the result is as expected (i.e., same as what you get when `num_processes=1`)? 

`test_q_score` looks like a good candidate for this, as there are two samples in there. I would copy that to a new function called something like `test_q_score_two_processes`, and (ideally) the only thing that needs to change is added that parameter to all calls to `q_score` in that test.